examples/optimize/optimize.py

```diff
@@ -62,7 +61,22 @@ def make_llm(cfg: DictConfig) -> LiteLLM:
         "presence_penalty": 0,
         "n": 1,
     }
-    return LiteLLM(model_name="gpt-3.5-turbo", parameters=parameters, use_cache=cfg.llm_cache)
+    if cfg.llm_name.startswith("gpt"):
+        llm = LiteLLM(model_name=cfg.llm_name, parameters=parameters, use_cache=cfg.llm_cache)
+    elif cfg.llm_name.startswith("meta-llama"):
+        # See model_name here: https://docs.together.ai/docs/serverless-models
+        # See corresponding tokenizer_name here: https://huggingface.co/meta-llama
+        llm = TrainableLLM(
```

Comment:
FYI, you can use `LiteLLM` as well for Together endpoints. Maybe we could avoid using `llm_name` configuration option if we go this way. Consider reusing some llm configs from `conf/llm`.